docs/dev/deploying/authentication_backends.rst

```diff
@@ -6,17 +6,17 @@ Connect does not have a built-in authentication system and instead relies on the
 
 .. _Python Social Auth: http://psa.matiasaguirre.net/
 
-NGPVAN ActionID
+NGP VAN ActionID
 ===============
```

Comment:
You'll want to add an additional `=` below to make the `=====` the same length as the headline